packages/SwingSet/src/kernel/liveSlots.js

```diff
@@ -936,7 +941,7 @@ function build(
     WeakSet: vom.VirtualObjectAwareWeakSet,
   });
 
-  function setBuildRootObject(buildRootObject) {
+  function setBuildRootObject(buildRootObject, vatID) {
```

Comment:
This `vatID` parameter (and its two new uses) should be removed in favor of `forVatID` (which was committed to earlier, when `build()` was called).

Reply:
Grrrr.  That's what I wanted and I couldn't find it...